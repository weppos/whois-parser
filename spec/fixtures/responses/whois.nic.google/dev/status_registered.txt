Domain Name: google.dev
Registry Domain ID: 2CF25B78E-DEV
Registrar WHOIS Server: whois.nic.google
Registrar URL: http://www.charlestonroadregistry.com
Updated Date: 2019-06-13T22:30:20Z
Creation Date: 2018-06-13T22:30:20Z
Registry Expiry Date: 2020-06-13T22:30:20Z
Registrar: Charleston Road Registry Billable
Registrar IANA ID: 9998
Registrar Abuse Contact Email: registry-abuse-support@google.com
Registrar Abuse Contact Phone: +1.2125652632
Domain Status: autoRenewPeriod https://icann.org/epp#autoRenewPeriod
Domain Status: ok https://icann.org/epp#ok
Registry Registrant ID: REDACTED FOR PRIVACY
Registrant Name: REDACTED FOR PRIVACY
Registrant Organization: Charleston Road Registry, Inc.
Registrant Street: REDACTED FOR PRIVACY
Registrant Street:
Registrant City: REDACTED FOR PRIVACY
Registrant State/Province: CA
Registrant Postal Code: REDACTED FOR PRIVACY
Registrant Country: US
Registrant Phone: REDACTED FOR PRIVACY
Registrant Fax: REDACTED FOR PRIVACY
Registrant Email: Please query the WHOIS server of the owning registrar identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name. 
Registry Admin ID: REDACTED FOR PRIVACY
Admin Name: REDACTED FOR PRIVACY
Admin Organization: REDACTED FOR PRIVACY
Admin Street: REDACTED FOR PRIVACY
Admin Street:
Admin City: REDACTED FOR PRIVACY
Admin State/Province: REDACTED FOR PRIVACY
Admin Postal Code: REDACTED FOR PRIVACY
Admin Country: REDACTED FOR PRIVACY
Admin Phone: REDACTED FOR PRIVACY
Admin Fax: REDACTED FOR PRIVACY
Admin Email: Please query the WHOIS server of the owning registrar identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name. 
Registry Tech ID: REDACTED FOR PRIVACY
Tech Name: REDACTED FOR PRIVACY
Tech Organization: REDACTED FOR PRIVACY
Tech Street: REDACTED FOR PRIVACY
Tech Street:
Tech City: REDACTED FOR PRIVACY
Tech State/Province: REDACTED FOR PRIVACY
Tech Postal Code: REDACTED FOR PRIVACY
Tech Country: REDACTED FOR PRIVACY
Tech Phone: REDACTED FOR PRIVACY
Tech Fax: REDACTED FOR PRIVACY
Tech Email: Please query the WHOIS server of the owning registrar identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name. 
Name Server: ns1.googledomains.com
Name Server: ns2.googledomains.com
Name Server: ns3.googledomains.com
Name Server: ns4.googledomains.com
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2019-06-28T20:29:01Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Please query the WHOIS server of the owning registrar identified in this
output for information on how to contact the Registrant, Admin, or Tech
contact of the queried domain name.

WHOIS information is provided by Charleston Road Registry Inc. (CRR) solely
for query-based, informational purposes. By querying our WHOIS database, you
are agreeing to comply with these terms
(https://www.registry.google/about/whois-disclaimer.html) and acknowledge
that your information will be used in accordance with CRR's Privacy Policy
(https://www.registry.google/about/privacy.html), so please read those
documents carefully.  Any information provided is "as is" without any
guarantee of accuracy. You may not use such information to (a) allow,
enable, or otherwise support the transmission of mass unsolicited,
commercial advertising or solicitations; (b) enable high volume, automated,
electronic processes that access the systems of CRR or any ICANN-Accredited
Registrar, except as reasonably necessary to register domain names or modify
existing registrations; or (c) engage in or support unlawful behavior. CRR
reserves the right to restrict or deny your access to the Whois database,
and may modify these terms at any time.
