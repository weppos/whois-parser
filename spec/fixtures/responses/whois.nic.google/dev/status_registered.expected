#domain
  %s == "google.dev"

#domain_id
  %s == "2CF25B78E-DEV"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2018-06-13T22:30:20Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2019-06-13T22:30:20Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2020-06-13T22:30:20Z}

#registrar
  %s %CLASS{registrar}
  %s.id           == "9998"
  %s.name         == "Charleston Road Registry Billable"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].organization  == "Charleston Road Registry, Inc."
  %s[0].state         == "CA"
  %s[0].country_code  == "US"

#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.googledomains.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.googledomains.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.googledomains.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.googledomains.com"
